src/basis/basis.js

```diff
@@ -309,16 +351,19 @@ export class Basis {
      * This guarantees the independence from the order in the elements array.
      * @return {String}
      */
-    getAsSortedString(inStandardRepresentation = true) {
+    getAsSortedString() {
         // make a copy to prevent modifying class values
         const clsInstance = new Basis(this.toJSON());
         clsInstance.toStandardRepresentation();
-        return clsInstance.elementsAndCoordinatesArray.map((entry) => {
-            const element = entry[0];
-            const coordinate = entry[1];
-            const toleratedCoordinates = coordinate.map(x => math.round(x, HASH_TOLERANCE));
-            return `${element} ${toleratedCoordinates.join()}`;
-        }).sort().join(';') + ";";
+        return `${clsInstance.elementsAndCoordinatesArray
+            .map((entry) => {
```

Comment:
Let's try and keep the first function call on the same line as the object reference?

Reply:
This closed issue https://github.com/eslint/eslint/issues/10977 almost discusses what we want. In general, I'm not sure it's possible to force in-lining the first in a chain of method calls. For this particular example, it can still be refactored for readability.

Reply:
Let's see if we can use the Airbnb example config from https://github.com/airbnb/javascript/tree/master/linters to support example 9.3 in https://github.com/airbnb/javascript/tree/master/linters

Reply:
I think this example is a pathological case due to the string interpolation. in `get atomicPositions` right below it it contains the style we prefer but of course it's not chained. Then in `hasEquivalentCellTo` as a chained call it is inconsistent again. The only way around it would be to sprinkle `// prettier-ignore` comments or remove `prettier` entirely and leave considerably style formatting up to the developer, with `eslint` doing what it can.